test.js

```diff
@@ -22,20 +25,24 @@ test('drive = multidrive', function (t) {
     t.throws(multidrive.bind(null, {}), /function/)
     t.throws(multidrive.bind(null, {}, noop), /function/)
   })
+  t.end()
 })
 
 test('drive.create', function (t) {
   t.test('should create an archive', function (t) {
-    t.plan(5)
+    t.plan(6)
 
     var store = toilet('state.json')
-    multidrive(store, createArchive, noop, function (err, drive) {
+    multidrive(store, createArchive, noopCb, function (err, drive) {
```

Comment:
this requirement to call the cb in `closeArchive` needs to be documented

Reply:
To be honest I am a bit puzzled about this request. `closeArchive` was always supposed to call the callback (that didn't change) the only thing that changes is that `closeArchive` is now needed in some tests in order to disconnect the drives. The only way I found this necessary to be documented was in the readme: https://github.com/datproject/multidrive/pull/14/commits/594dc3147e79ad8b5caa7d827f05c0c8a37549d8